dbt/adapters/trino/connections.py

```diff
@@ -103,6 +103,7 @@ class TrinoNoneCredentials(TrinoCredentials):
     http_headers: Optional[Dict[str, str]] = None
     session_properties: Dict[str, Any] = field(default_factory=dict)
     prepared_statements_enabled: bool = PREPARED_STATEMENTS_ENABLED_DEFAULT
+    retries: Optional[int] = trino.constants.DEFAULT_MAX_ATTEMPTS
```

Comment:
shouldn't this setting be optional?

Reply:
This setting is indeed typed optional, we could have put it to None instead of using the Trino constant